### PR TITLE
feat #112 : Create an The About us Page

### LINKED ISSUE #112: build an about us page with the repository contribution list
Currently, there's no section on our website that highlights the efforts and contributions of our repository contributors. This lack of visibility makes it difficult for new visitors to appreciate the collaborative efforts behind the project and see the history of contributions made by various team members.

please assign me this issue i want to work on this 

 example
image

### PR DESCRIPTION
**Pull request by CoderSwarup**
close #112 

build the about us page that show the details about the project and the contributors list of the project
ui screen short 
image

### CODE REVIEW

**Comment by afeefuddin** [comment]
We use modular css in our project. Please rename the CSS files to `.module.css` and do imports accordingly.

**Comment by CoderSwarup** [comment]
@afeefuddin  check now

**Comment by afeefuddin** [comment]
Squash both the commits.

**Comment by CoderSwarup** [comment]
its correct

**Comment by subharthihazra** [comment]
in Aboutus and ContributorCard page change the import of CSS to ".module.css". The files are renamed but the imports are not.

**Comment by CoderSwarup** [comment]
@subharthihazra please check now

**Comment by CoderSwarup** [comment]
hey @subharthihazra  any update

**Comment by afeefuddin** [comment]
Hey @CoderSwarup, Ideally you should not tag the maintainers unnecessarily, we'll check the PR when we are free.

**Comment by afeefuddin** [comment]
LGTM overall. There is a small bug related to <Link /> of react-router dom. It is not scrolling to top on the page change. Can you fix that @CoderSwarup? Once it is fixed the PR is ready to merge. Please ensure to squash all the commits that you will make.

**Comment by CoderSwarup** [comment]
I solve this please check it now

File: Frontend/src/pages/AboutUs/AboutPage.jsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        console.error("Error fetching contributors:", error);
+      }
+    }
+
+    fetchContributors();
+  }, []);
+
+  useEffect(() => {
+    window.scrollTo({ top: 0, behavior: "instant" });
+  });
```
Comment (afeefuddin) [inline Frontend/src/pages/AboutUs/AboutPage.jsx:26]: Why there is no empty dependency array here? This effect will run every time there is a state change.

Comment (afeefuddin) [reply Frontend/src/pages/AboutUs/AboutPage.jsx:26]: Can you explain to me why this code is correct, if it is?

Comment (CoderSwarup) [reply Frontend/src/pages/AboutUs/AboutPage.jsx]: sorry for this

**Comment by afeefuddin** [comment]
Merged, thanks for doing this @CoderSwarup

**Pull request merged**